pkg/controller/controller_binding_test.go

```diff
@@ -4066,6 +4064,131 @@ func TestTransformSecretData(t *testing.T) {
 	}
 }
 
+// TestReconcileServiceBindingUserProvided tests reconcile binding
+// to ensure bindings that reference an instance with UserProvided
+// set and no class/plan work
+func TestReconcileServiceBindingUserProvided(t *testing.T) {
```

Comment:
Do we have a test for a UPS that has empty creds?  Meaning no auth is required to talk to the service.